lib/linguist/vendor.yml

```diff
@@ -224,19 +228,6 @@
 # Html5shiv
 - (^|/)html5shiv\.js$
 
-# Samples folders
-- ^[Ss]amples/
-
-# LICENSE, README, git config files
-- ^COPYING$
-- LICENSE$
-- License$
```

Comment:
I think "License" won't be matched by `documentation.yml`. Can you add a test for it, and fix it if the test fails?